examples/secrets/README.md

```diff
@@ -1,16 +1,27 @@
 # Using secrets
 
-Using secret is similar to any other pod. As of now this does not provide a way to create a secret but only to consume it.
-
 Creating secret:
 
-```bash
-oc create secret generic wordpress --from-literal='MYSQL_ROOT_PASSWORD=rootpasswd,DB_PASSWD=wordpress'
+Create a secret by defining it at the root level -
+```yaml
+secrets:
+- name: wordpress
+  data:
+    MYSQL_ROOT_PASSWORD: YWRtaW4=
+    MYSQL_PASSWORD: cGFzc3dvcmQ=
```

Comment:
Most important thing is to explain this in file-reference not in the example. 

Examples shouldn't supply documentation.